src/main/resources/db/migration/postgresql/V1_10__2055_Topic_Column_On_Workstack_Team_Queue.sql

```diff
@@ -0,0 +1,16 @@
+DROP VIEW IF EXISTS stage_data;
```

Comment:
Do we want to get rid of all exiting stage data?

Reply:
As I understand views they are virtual table/composition table created from "real tables", and don't hold the actual data, so wouldn't be removing real data, just the composition of the view? I added this in when I added the alias which was recommended by @LiamMacP , as without it the query wouldn't run. Happy to alter if there is a better way of achieving this though.

Reply:
I'll try it without

Reply:
Yeah `stage_data` is a bit of a combination of data, but records from `stage_data` are retrieved from the frontend on specific actions. So I'm thinking if we we get rid of this view, actions for existing cases will break as it will no longer have the historical data it needs for some of the queries.

Reply:
See `hocs-casework/src/main/java/uk/gov/digital/ho/hocs/casework/domain/repository/StageRepository.java` for examples of where we rely on data in `stage_data`

Reply:
So without it the application fails to run due to:
`2021-01-12 12:09:19.979  WARN 49767 --- [           main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flywayInitializer' defined in class path resource [org/springframework/boot/autoconfigure/flyway/FlywayAutoConfiguration$FlywayConfiguration.class]: Invocation of init method failed; nested exception is org.flywaydb.core.internal.command.DbMigrate$FlywayMigrateException: 
Migration V1_10__2055_Topic_Column_On_Workstack_Team_Queue.sql failed
---------------------------------------------------------------------
SQL State  : 42P16
Error Code : 0
Message    : ERROR: cannot change name of view column "correspondents" to "case_assigned_topic"
Location   : db/migration/postgresql/V1_10__2055_Topic_Column_On_Workstack_Team_Queue.sql (/Users/robertdeniszczyc/repos/hocs-casework/build/resources/main/db/migration/postgresql/V1_10__2055_Topic_Column_On_Workstack_Team_Queue.sql)
Line       : 1
Statement  : CREATE OR REPLACE VIEW stage_data AS
    SELECT c.reference AS case_reference, c.type AS case_type, c.data as data, c.created AS case_created, s.*, t.text AS case_assigned_topic
    FROM stage s
            JOIN active_case c ON s.case_uuid = c.uuid
            LEFT JOIN casework.topic t ON c.primary_topic_uuid = t.uuid

2021-01-12 12:09:20.009 ERROR 49767 --- [           main] o.s.boot.SpringApplication               : Application run failed`

cc @LiamMacP as it mentions the correspondents stuff

Reply:
This has been removed.